### PR TITLE
SV Bluntsmoken - God's gift to spess

### PR DESCRIPTION
**Pull request by DrDuckedGoose**
## About The Pull Request
A new shiptest featuring a department focused design. ~~SV Bluntsmoken~~ The 'Botanical-class' is a shiptest centered around botany, including some elements from traditional shiptests and incorporations from botany-ruins such as the seed vault.
image
image

No, you can't get roundstart gatfruit. Pay for it, nerd.

## Why It's Good For The Game
This PR aims to offer a relaxing experience for players who find the normal gameplay loop 'too much for the moment.' Additionally, the following diagram promotes an understanding of the concept; [link](https://www.google.com/url?sa=i&url=https%3A%2F%2Ftwitter.com%2Fpicswithutost%2Fstatus%2F1443419032707866626&psig=AOvVaw0LGuWcuXhdJE9BhaeO7g5W&ust=1641488348113000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCPDR0uGKm_UCFQAAAAAdAAAAABAU)

## Changelog
:cl:
add: Added btn_satellite.dmm
add: Added sv_bluntsmoken.json
add: Added special contraband category to biogenerator designs
add: Added alternate biogenerator with access to contraband category. 
/:cl:

### CODE REVIEW

**Comment by AtlasRules** [comment]
NO LITTLE SPESSMAN, DON'T GO INTO THE WEEDSHIP

**Comment by Dimasw99** [comment]
SV Bluetenrauchen

**Comment by Zevotech** [comment]
gatfruit my beloved

**Comment by triplezeta** [comment]
this is pretty and all but do we really need more ships

File: _maps/configs/sv_bluntsmoken.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"map_name": "SV Bluntsmoken",
+	"map_short_name": "Weed ship",
```
Comment (triplezeta) [inline _maps/configs/sv_bluntsmoken.json]: follow `[X]-class [Purpose] Vessel` name format for `map_name` and `[X]-class` name format for `map_short_name`

**Comment by DrDuckedGoose** [comment]
> this is pretty and all but do we really need more ships

Believe it or not, regardless of what I put out, I'm not being paid for it. If I don't like what I'm making, chance is I won't go out of my way to make it.

**Comment by triplezeta** [comment]
i get that but we have over 30 ships in rotation with 3 of them intended to cover the botany niche

**Comment by DrDuckedGoose** [comment]
I personally don't find them adequate. However, I can certainly understand the impression of oversaturation.

**Comment by PrefabQuasar** [comment]
while i like the ship, the name is probably a little bit too on the nose
edit: ive been informed the name has already been changed, disregard

**Comment by PrefabQuasar** [comment]
hol up this ship has roundstart gatfruit? that may be a bit overpowered for a shiptest

**Comment by DrDuckedGoose** [comment]
> hol up this ship has roundstart gatfruit? that may be a bit overpowered for a shiptest

I might make some custom changes to the biogenerator that allows you to purchase a pack for an aggressively high price. Specifically for this map, however.

**Comment by UltraNumeron** [comment]
no more 100 revolvers 30 mins into round but yeah, one more thing is that the dna manip is pretty hard capped with no higher tier parts, but that isnt too much of a problem when you can overdose them with botany chems.

**Comment by PrismPipo** [comment]
APC electrical system is missing a vertical wire under the door to both EVA and Crew Quarters, under catwalks under doors
Capture2
capture3
.

**Comment by PrismPipo** [comment]
> APC electrical system is missing a vertical wire under the door to both EVA and Crew Quarters, under catwalks under doors Capture2 capture3 .

the green is what we added

**Comment by PrismPipo** [comment]
> > APC electrical system is missing a vertical wire under the door to both EVA and Crew Quarters, under catwalks under doors Capture2 capture3 .
> 
> the green is what we added

heck, also there's not a wideband on the bridge normally, one had to be added during round

**Comment by DrDuckedGoose** [comment]
How & where are you playing this ship dawg?

**Comment by miniusAreas** [comment]
from Shiptest Shiplist Review, in #shipyard

Bluntsmoken/Botanical-Class

> Smoke weed, make drugs, do crime. Otherwise, Doesn’t need an ore silo period, and the ORM is pushing it hard. Also, kinda lacking in the decal or detail department.

Small feedback to pick your brain.

File: code/modules/research/designs/biogenerator_designs.dm [hunk 2]
```diff
@@ -169,3 +169,19 @@
...
+	build_path = /obj/item/seeds/gatfruit
+	category = list("initial","Contraband")
+
+/datum/design/cherrybomb
+	name = "Cherrybomb Seeds"
+	id = "cherrybomb_seeds"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass= 100000)
+	build_path = /obj/item/seeds/cherry/bomb
+	category = list("initial","Contraband")
```
Comment (MarkSuckerberg) [inline code/modules/research/designs/biogenerator_designs.dm]:
```suggestion
	category = list("initial","Contraband")

```

**Comment by DrDuckedGoose** [comment]
ORM removed btw

**Pull request closed without merging**